newIDE/app/src/Utils/GDevelopServices/Asset.js

```diff
@@ -232,3 +232,11 @@ export const listAllLicenses = (): Promise<Array<License>> => {
     })
     .then(response => response.data);
 };
+
+export const isPixelArt = (assetShortHeader: AssetShortHeader) => {
```

Comment:
This can be a single instruction:

Return assetShortHeader.tags.some(tag => ...

No need for extra Boolean